plugin/src/test/java/org/opensearch/ml/action/connector/UpdateConnectorTransportActionTests.java

```diff
@@ -14,10 +14,8 @@
 
 import java.io.IOException;
 import java.nio.file.Path;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+import java.time.Instant;
+import java.util.*;
```

Comment:
It's generally frowned on to use star imports.

Reply:
oops, yeah will fix that